orchestrator-salt/src/main/resources/salt/salt/datalake_resize/check_atlas_updated.sls

```diff
@@ -0,0 +1,8 @@
+include:
+  - datalake_resize.check_atlas_updated
+
+check_atlas_updated:
+  cmd.run:
+    - name: /opt/salt/scripts/check_atlas_updated.sh {{salt['pillar.get']('check_atlas_updated:max_retries')}}
+    - require:
+      - sls: datalake_resize.check_atlas_updated
```

Comment:
this doesn't seem right. what are your intentions here?

Reply:
The intention here is to pass in the pillar file into this salt file so the argument can be used.

Reply:
you would need the require part for the file and not the state

Reply:
What do you mean by that? I see some requires for files like bash files and the like, but I want to use the variable from the pillar file `atlas/check_atlas_updated.sls`.

I am effectively trying to do the same as done here: https://github.com/hortonworks/cloudbreak/blob/master/orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/backup.sls 
where they use values from the `postgresql/disaster_recovery.sls` pillar file.